src/UI/Components/InlineEditable/EditableField.tsx

```diff
@@ -45,29 +48,26 @@ export const EditableField: React.FC<Props> = ({
   EditView,
   StaticView,
   alignActions,
+  error,
+  setError,
 }) => {
   const alignment = alignActions === "end" ? "alignSelfFlexEnd" : "alignSelfFlexStart";
   const [editable, setEditable] = useState(initiallyEditable);
-  const [submitError, setSubmitError] = useState("");
   const [value, setValue] = useState(initialValue);
   const onSubmitRequest = async (value: string) => {
     setEditable(false);
-    const error = await onSubmit(value);
-
-    if (Maybe.isSome(error)) {
-      setSubmitError(error.value);
-    }
+    onSubmit(value);
   };
   const onEditClick = () => {
     setEditable(true);
-    setSubmitError("");
+    setError(null);
```

Comment:
does this mean that whenever the user clicks on the field, the error gets cleared?

Reply:
Yes, as it was so far

Reply:
Can you make a ticket to review this behaviour? I'm not entirely certain this is UX friendly

Reply:
https://github.com/inmanta/web-console/issues/6330